packages/react-native/React/Views/ScrollView/RCTScrollView.h

```diff
@@ -50,6 +50,8 @@
 @property (nonatomic, assign) BOOL inverted;
 /** Focus area of newly-activated text input relative to the window to compare against UIKeyboardFrameBegin/End */
 @property (nonatomic, assign) CGRect firstResponderFocus;
+/** newly-activated text input relative to the window to compare against UIKeyboardFrameBegin/End */
+@property (nonatomic, weak) UIView *firstResponderFocusView;
```

Comment:
There's a lot of overlap between these two fields. If we're going to be storing the focusView anyway, can we get rid of the focus rect and instead add a `reactFocusFrame` to `UIView+React`?